ouroboros-consensus/tools/db-analyse/Main.hs

```diff
@@ -180,6 +183,47 @@ showBlockHeaderSize immDB epochInfo rr = do
         Chain.ABlock { blockHeader } = block
         blockHeaderSz = Chain.headerLength blockHeader
 
+{-------------------------------------------------------------------------------
+  Analysis: show the total transaction sizes in bytes per block
+-------------------------------------------------------------------------------}
+
+showBlockTxsSize :: Analysis
+showBlockTxsSize immDB epochInfo rr = processAll immDB rr go
+  where
+    go :: ByronBlock -> IO ()
+    go blk =
+        case blk of
+          Byron.ByronBlock (Chain.ABOBBlock regularBlk) _ _ ->
+            go' (blockSlot blk) regularBlk
+          _otherwise ->
+            return () -- Skip EBBs
+
+    go' :: SlotNo -> Chain.ABlock ByteString -> IO ()
```

Comment:
Same remark about `go` and `go'`. I guess renaming here is the way to go.

Alternative vague idea: make an Analysis modifier/combinator that only applied an analysis to non-EBB blocks.